frontend/jest.setup.ts

```
@@ -309,6 +335,6 @@
 // Clean up after each test
 afterEach(() => {
   jest.clearAllMocks();
-  delete global.mockSearchParams;
-  delete global.mockPathname;
+  delete (global as any).mockSearchParams;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)